draft-ietf-tls-tls13.md

```diff
@@ -1272,6 +1265,53 @@ The key exchange messages are used to exchange security capabilities
 between the client and server and to establish the traffic keys used to protect
 the handshake and data.
 
+
+### Cryptographic Negotiation
+
+TLS cryptographic negotiation proceeds by the client offering the
+following four sets of options in its ClientHello.
+
+- A list of cipher suites which indicates the AEAD cipher/HKDF hash
+  pairs which the client supports
+- A "supported_group" ({{negotiated-groups}}) extension which indicates the (EC)DHE groups
+  which the client supports and a "key_share" ({{key-share}}) extension which contains
+  (EC)DHE shares for some or all of these groups
+- A "signature_algorithms" ({{signature-algorithms}}) extension which indicates the signature
+  algorithms which the client can accept.
+- A "pre_shared_key" ({{pre-shared-key-extension}}) extension which contains the identities
+  of symmetric keys known to the client and the key exchange
+  modes which each PSK supports.
+
+If the server does not select a PSK, then the first three of these
+options are entirely orthogonal: the server independently selects a
+cipher suite, an (EC)DHE group and key share for key establishment,
+and a signature algorithm/certificate pair to authenticate itself.
+If there is no overlap on any of these parameters, then the handshake
```

Comment:
"If any of these parameters has no overlap between the values supported by the client and server" (the current text could be misread as requiring all parameters to have no overlap before handshake failure).